cdap-ui/app/cdap/components/PipelineDetails/ProfilesListView/ProfileCustomizePopover/index.js

```diff
@@ -51,7 +51,7 @@ export default class ProfileCustomizePopover extends PureComponent {
 
   render() {
     let {name, provisioner, scope} = this.props.profile;
-    let profileName = scope === 'SYSTEM' ? `system:${name}` : `user:${name}`;
+    let profileName = scope === 'SYSTEM' ? `SYSTEM:${name}` : `USER:${name}`;
```

Comment:
nit: Just like `extractProfileName` we may also want to extract this out as a function. Given the scope return the right profile name.